src/render/solver.rs

```diff
@@ -35,8 +35,7 @@ fn print_as_table(packages: &[RepoDataRecord]) {
             package
                 .channel
                 .as_ref()
-                .map(|s| s.rsplit('/').find(|s| !s.is_empty()))
-                .flatten()
+                .and_then(|s| s.rsplit('/').find(|s| !s.is_empty()))
```

Comment:
clippy complained about it

Reply:
yeah i made the same change a few days ago but maybe it's not merged yet.